scripts/build.sh

```diff
@@ -0,0 +1,15 @@
+OPTIMIZER_VER=0.12.6
+
+if [[ `uname -m` =~ "arm" ]]; then 
+OPTIMIZER_IMG=cosmwasm/workspace-optimizer-arm64; else 
```

Comment:
careful with arm64 and rustc wasm builds. 

> do NOT run .wasm's built on native arm64 in production. i repeat. use the qemu x64 normal docker image. this is advise from ethan frey / other core devs @ cosmwasm

Reply:
It just for development purpose is not for production.